app/services/diffed_content.rb

```diff
@@ -0,0 +1,53 @@
+class DiffedContent
+  attr_reader :source, :target
+
+  def initialize(source, target)
+    @source = source
+    @target = target
+
+    normalize_content(@source)
+    normalize_content(@target)
+  end
+
+  def diff
+    Differ.format = :html
+    differ_result = Differ.diff_by_word(source.content, target.content)
```

Comment:
Why do we need a custom format rather than using the existing approach we have for Revision history?

https://github.com/dradis/dradis-ce/blob/develop/app/models/diffed_revision.rb#L12-L16